sematic/api/endpoints/tests/BUILD

```diff
@@ -1,6 +1,7 @@
 pytest_test(
     name = "test_external_resources",
     srcs = ["test_external_resources.py"],
+    py_versions = [PY3.PY3_9, PY3.PY3_10, PY3.PY3_11],
```

Comment:
Just removing testing of python3.8 from certain tests that are giving some trouble with python3.8, not because of issues with the code they are testing, but rather with the tests themselves. Not sure why this is showing up now, but dropping 3.8 support is imminent as it has reached EOL.